js/index.js

```diff
@@ -1,7 +1,55 @@
-// Iteration 1: Names and Input
+console.log("I'm ready!");
+
+let hacker1 = "Tim"
+console.log("The driver's name is " + hacker1);
+
+let hacker2 = "Saeka"
+ console.log("The navigator's name is " + hacker2);
+
+if  (hacker1.length > hacker2.length) {
+console.log("The driver has the longest name, it has " + hacker1.length + " characters")
+} else if (hacker1.length < hacker2.length) {
+  console.log("It seems that the navigator has the longest name, it has " + hacker2.length + " characters")
+} else {
+  console.log("wow, you both have equally long names, " + hacker2.length + " characters!")
+}
+
+let driverUpperCase = hacker1.toUpperCase();
+console.log(driverUpperCase[0] + " "+ driverUpperCase[1]+ " "+driverUpperCase[2]);
```

Comment:
Writing it this way will not allow your code to be flexible. This means that if your hacker1 changes his name to more than three letters, the other letters will be upperCase but will not have a space between them. 

What you can do is to loop over the "driverUpperCase" string and then add a space in every iteration. Another way (and this is much simpler) is to use the .split() and .join() method. 

In this case, you would write:

`driverUpperCase.split("").join(" ")` --> meaning you want to **split** the string at every character. Then this creates an array that you want to **join** with space.